ios/REAModule.mm

```diff
@@ -51,23 +51,25 @@ @implementation REAModule {
 
 RCT_EXPORT_MODULE(ReanimatedModule);
 
+#ifdef DEBUG
 // This counts how many instances of REAModule are present. Thanks to this we
 // can easily detect if there is a memory leak (ex. due to a retain cycle).
-static int instanceCounter = 0;
+static int instanceCounterREAModule = 0;
 
 - (id)init
 {
   // The counter may be 1 during a reload, because the previous instance
   // may not have been deallocated in time, but it should not be higher.
-  NSAssert(instanceCounter <= 1, @"More than one REAModule instance present");
-  instanceCounter++;
+  NSAssert(instanceCounterREAModule <= 1, @"More than one REAModule instance present");
```

Comment:
```suggestion
  NSAssert(instanceCounterREAModule <= 1, @"More than one REAModule instance present. This may indicate a memory leak due to a retain cycle.");
```